docs/DeveloperGuide.md

```diff
@@ -347,17 +347,29 @@ Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (un
 Priority | As a ... | I want to ... | So that I can...
 -------- | :-------- | :--------- | :-----------
 `* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
-`* * *` | user | add a new person |
-`* * *` | user | delete a person | remove entries that I no longer need
-`* * *` | user | find a person by name | locate details of persons without having to go through the entire list
-`* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
-`*` | user with many persons in the address book | sort persons by name | locate a person easily
-
-{More to be added}
+`* * *` | user | add a new task with completion date | remind myself of things I have to do and the deadlines to meet
+`* * *` | user | add a new [recurring task](#recurring-task) | prevent the need of keying the same cyclical task every period
+`* * *` | user | see all tasks | have an overview of all the things I need to do and decide what I should do first
+`* * *` | user | update a task | change entries that are errorneous or outdated
```

Comment:
Oh and can you elaborate on what is floating task? 
I will add in CRUD of events, it should have same structure as tasks.